README.md

```diff
@@ -35,18 +49,73 @@ poetry install
 To use GPU, the correct `torch` and `torch_geometric` version should be installed depending on your host device. You may first need to `pip uninstall` these packages, provided by requirements.
 Then follow installation instructions provided on [torch](https://pytorch.org/get-started/locally/) and [torch_geometric](https://pytorch-geometric.readthedocs.io/en/stable/install/installation.html#installation-from-wheels) websites.
 
+## Repo structure
+
+- **AIAgent** contains Python agent and related infrastructure to train network, prepare data, etc.
+- **GameServers** contains (as submodules) different symbolic execution engines extended to communicate with the agent, generate data for training, etc.
+- **maps** contains target projects that used as inputs for symbolic execution engines, as data for training.
```

Comment:
```suggestion - **maps** contains target projects that are used as inputs for symbolic execution engines as training data.
```

Reply:
Why should we remove this? I think it's important part of our repository structure.

Reply:
this suggestion was an accident